src/components/HookForm/index.jsx

```diff
@@ -508,61 +531,85 @@ export default class HookForm extends Component {
-                  />
-                </ListItem>
-              )}
-              <ListItem>
+          <ListItem>
+            {hookLastFires ? (
+              <List className={classes.hookLastFires}>
                 <ListItemText
-                  primary="Next Scheduled Fire"
-                  secondary={
-                    hook.status.nextScheduledDate ? (
-                      <DateDistance from={hook.status.nextScheduledDate} />
-                    ) : (
-                      'n/a'
-                    )
+                  primary={
+                    <Typography variant="subtitle1">
+                      Last Hook Fire Attempts
+                    </Typography>
                   }
                 />
-              </ListItem>
-            </Fragment>
+                {hookLastFires.map(
+                  ({ taskId, taskCreateTime, firedBy, result, error }) => (
+                    <ListItem key={taskId}>
+                      <Grid container direction="column">
+                        <Grid
+                          item
+                          container
+                          direction="row"
+                          justify="space-between">
+                          <Grid item>
+                            <pre>{firedBy}</pre>
+                          </Grid>
+                          <Grid item>
+                            <pre>
+                              <DateDistance from={taskCreateTime} />
+                            </pre>
+                          </Grid>
+                        </Grid>
+                        <Grid item>
+                          <ExpansionPanel
+                            expanded={
+                              result === 'ERROR' &&
+                              lastFirePanelExpanded === taskId
+                            }
+                            onChange={this.handleLastFirePanelChange(taskId)}>
+                            <ExpansionPanelSummary
+                              expandIcon={error && <ExpandMoreIcon />}>
+                              <Grid container justify="space-between">
+                                <Grid item>
+                                  {(result === 'SUCCESS' && (
+                                    <Link to={`/tasks/${taskId}`}>
+                                      <Typography>{taskId}</Typography>
+                                    </Link>
+                                  )) || <Typography>{taskId}</Typography>}
+                                </Grid>
+                                <Grid item>
+                                  <Label mini status={result.toLowerCase()}>
+                                    {result}
+                                  </Label>
+                                </Grid>
+                              </Grid>
+                            </ExpansionPanelSummary>
+                            <ExpansionPanelDetails>
+                              <ErrorPanel error={error} onClose={null} />
```

Comment:
We could probably have errors be a separate column in the table and use simple text to display the error instead of having it in a error panel. We use error panels for when the server fails to return data or the user is trying to do something not allowed.